src/components/Card.vue

```diff
@@ -171,13 +171,7 @@ export default {
     border-radius: $big-border-radius;
 
     &:hover {
-      box-shadow: 0 5px 10px var(--color-card-shadow);
       transform: scale(1.007);
-
-      @media (prefers-reduced-motion: reduce) {
-        box-shadow: none;
-        transform: none;
```

Comment:
We should leave this line and just remove the box-shadow part, right?

Reply:
Yes, totally, my mistake!